packages/graphql-armor/src/apollo/armor.ts

```diff
@@ -24,26 +27,38 @@ export class ApolloArmor {
   }
 
   protect(): {
-    plugins: PluginDefinition[];
+    plugins: any;
     validationRules: ValidationRule[];
     allowBatchedHttpRequests: false;
-    debug: false;
+    includeStacktraceInErrorResponses: false;
```

Comment:
I think it can be moved over v4 only

`debug` remains false in v3 and `includeStacktraceInErrorResponses` only in v4